src/app/menu/menu.component.scss

```diff
@@ -0,0 +1,223 @@
+    position: absolute;
+    left: $outer-circle-left-pos;
+    border: 1px solid grey;  
+}
+
+.intro-outer-circle {
+    @extend .outer-circle;
+    background: #fff;  
+    top: $outer-circle-top-pos;     
+}
+
+.inner-circle {
+    width: 52px;
+    height: 52px;
+    border-radius: 50px;
+    font-size: 20px;
+    font-weight: 900;
+    color: #fff;
+    line-height: 50px;
+    text-align: center;
+    position: absolute;
+    left: 17%;
+    top: 18%;
+    border: 1px solid grey;  
+}
+
+.intro-inner-circle {
+    @extend .inner-circle;
+    background: #00F977;
+}
+
+.malaria-inner-circle {
+    @extend .inner-circle;
+    background: #EB5A61;
+}
+
+.med-inner-circle {
+    @extend .inner-circle;
+    background: #017E84;
+}
+
+.pc-policy-inner-circle {
+    @extend .inner-circle;
+    background: #F2AE3E;
+}
+
+.malaria-outer-circle {
+    @extend .outer-circle;
+    background: #E13946;  
+    top: $outer-circle-top-pos+20; 
+}
+
+.med-outer-circle {
+    @extend .outer-circle;
+    background: #07717E;
+    top: $outer-circle-top-pos+40;
+}
+
+.pc-policy-outer-circle {
+    @extend .outer-circle;
+    background: #F09938;
+    top: $outer-circle-top-pos+60;
+}
+
+.unlocked-outer-circle {
+    @extend .outer-circle;
+    background: #fff;
+    top: $outer-circle-top-pos+80;
+}
+
+.activity {
+    border-radius: 200px;
+    border: 6px solid transparent;
+    height: 75px;
+    width: 75px;
+    position: absolute;  
+}
+
+.activity1 {
+    @extend .activity;
+    left: 3%;
+    top: 7%;    
+}
+
+.activity1-incomplete {
+    @extend .activity1;
+    border-top-color: #BEBEBE;
+}
+
+.activity1-complete {
+    @extend .activity1;
+    border-top-color: #00F977;    
+}
+
+.activity2 {
+    @extend .activity;
+    left: -2%;
+    top: 3%; 
+    -ms-transform: rotate(25deg); /* IE 9 */
+    -webkit-transform: rotate(25deg); /* Chrome, Safari, Opera */
+    transform: rotate(25deg);  
+}
+
+.activity2-incomplete {
+    @extend .activity2;
+    border-right-color: #BEBEBE;
+}
+
+.activity2-complete {
+    @extend .activity2;
+    border-right-color: #00F977;   
+}
+
+.activity3 {
+    @extend .activity;
+    left: 5%;
+    top: 3%; 
+    -ms-transform: rotate(-25deg); /* IE 9 */
+    -webkit-transform: rotate(-25deg); /* Chrome, Safari, Opera */
+    transform: rotate(-25deg);  
+}
+
+.activity3-incomplete {
+    @extend .activity3;
+    border-left-color: #BEBEBE; 
+}
+
+.activity3-complete {  
+    @extend .activity3;
+    border-left-color: #00F977;
+}
+
+.hex {
+    width: auto;
+    height: 20%;
+    position: absolute;
+    left: $hex-circle-left-pos;  
+}
+
+
+.intro-hex {
+    @extend .hex;  
+    top: $hex-circle-top-pos;
+}
+
+.malaria-hex {
+    @extend .hex;  
+    top: $hex-circle-top-pos+20;
+}
+
+.meds-hex {
+    @extend .hex;  
+    top: $hex-circle-top-pos+40;
+}
+
+.pc-policy-hex {
+    @extend .hex;  
+    top: $hex-circle-top-pos+60;
+}
+
+.unlocked-hex {
+    @extend .hex;  
+    top: $hex-circle-top-pos+80;
+}
+
+.btn {
```

Comment:
i don't like the idea of overriding bootstrap classes. can we give it another name instead?